scss/partials/_blog.scss

```diff
@@ -0,0 +1,404 @@
+                font-size: 2rem;
+                font-weight: 500;
+                list-style-type: none;
+                padding-bottom: 1.5rem;
+
+                &:last-child {
+                    padding-bottom: 0;
+                }
+
+            }
+
+            &--catagories {
+                padding-left: 0;
+                padding-bottom: 146px;
+
+                .blog-article__post-list-item {
+                    display: inline-block;
+                    margin-right: 0.8rem;
+
+                    &::before {
+                        content: '';
+                    }
+
+                    .button--small-secondary-flex {
+                        @extend .tk-degular-text;
+                        font-weight: 500;
+                        letter-spacing: -0.2px;
+                        line-height: 0.5;
+                        text-transform: capitalize;
+                    }
+                }
+            }
+
+        }
+
+    }
+}
+
+// Blog Author Footer
+.blog-author-footer {
+    margin: 0;
+    padding-top: 0;
+    width: 45.5rem;
+
+    @include breakpoint(desktop) {
+        margin: 0 auto;
+        width: 100%;
+    }
+
+    @media only screen and (min-width: 866px) and (max-width: 1160px) {
+        // TODO discuss with Mark and the Web team about how to handle custom breakpoints
+        margin: 0 auto;
+        width: 100%;
+    }
+
+    &-social {
+        @extend .blog-post-hero__time-social;
+        border-bottom: 0.1rem solid $cool-neutral-4;
+        border-top: 0;
+
+        &-date-time {
+            width: 86%;
+            @extend .blog-post-hero__time-social-read-time;
+
+            &--date-text,
+            &--date-format {
+                display: inline-block;
+            }
+            @include breakpoint(desktop) {
+                &--date-text { 
+                    margin-bottom: 1.6rem !important;
+                }
+            }
+            
+
+            &--date-format {
+                @extend .tk-degular-text;
+                color: $secondary-green-1;
+            }
+        }
+
+        &-media-links {
+            @extend .blog-post-hero__time-social-links;
+
+            .social-icon__link {
+                img {
+                    background-color: $secondary-blue-2;
+                }
+            }
+        }
+    }
+
+    &-bio {
+        display: grid;
+        align-items: center;
+        padding-bottom: 14.5rem;
+        padding-top: 7.6rem;
+
+        @include breakpoint(desktop) {
+            grid-template-columns: 1fr 2fr;
+        }
+
+        @media only screen and (min-width: 866px) and (max-width: 1160px) {
+            // TODO discuss with Mark and the Web team about how to handle custom breakpoints
+            grid-template-columns: 1fr;
+        }
+
+
+        border-bottom: 0.1rem solid $cool-neutral-4;
+        grid-column-gap: 8rem;
+
+        &__img-container {
+
+            margin-bottom: 3rem;
+
+            &__author-image {
```

Comment:
While I think I prefer this as our approach, we might want to just verify if images will be supplied already rounded or not. We might need to also consider what we do with all the existing authors in the system that have no headshot.